src/App.css

```diff
@@ -1,6 +1,9 @@
 body {
   background-color: #282c34;
   color: aliceblue;
+}
+
+.App {
```

Comment:
ideally css class names are supposed to be lowercase, unless this is some react builtin thing ?

Reply:
it was a default class when u create react app,, ill change it now

Reply:
then its fine if its built-in, i dont use react so wasnt sure